frontend/src/styles.scss

```diff
@@ -15,6 +15,53 @@ $fa-font-path: '../fonts';
   --menu-height: 35px;
 }
 
+.btnBox {
+  padding-bottom: 0.2rem;
+}
+
+.boxTable {
+  .alert-warning {
+    border-bottom-right-radius: 1rem;
+    border-bottom-left-radius: 1rem;
+  }
+}
+
+// customization of the button of datepicker
+  .datepicker-toggle {
+    border-top-right-radius: 1rem;
+    border-bottom-right-radius: 1rem;
+    background-color: $yellow;
+    color: $darkGrey;
+    border: 0px;
+      &:hover {
+        border-top-right-radius: 1rem;
+        border-bottom-right-radius: 1rem;
+        background-color: $electricBlue;
+        color: $white;
+        border: 0px;
+      }
+  }
+// change to yellow when hover a day in the ngb-datepicker
+.dropdown-menu.show {
+  .btn-light:hover {
+    background-color: $yellow;
+  }
+}
+.ngb-dp-weekday.small {
+  color: $electricBlue;
+}
+
+// container of the About screen
+#about {
+  border-radius: 1rem;
+}
```

Comment:
If specific for about component, it could be better to place it in the component's CSS